site/en/docs/android/custom-tabs/guide-interactivity/index.md

```diff
@@ -0,0 +1,197 @@
+        Intent shareIntent = Intent.createChooser(sendIntent, null);
+        context.startActivity(shareIntent);
+    }
+}
+```
+
+Now, create a PendingIntent for ShareBroadcast and register it via [`setActionButton`](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder#setActionButton(android.graphics.Bitmap,java.lang.String,android.app.PendingIntent)). Pass the pending intent together with the icon and the description. 
+
+```java
+String shareDescription = getString(R.string.label_action_share);
+Bitmap shareIcon = BitmapFactory.decodeResource(getResources(),
+       android.R.drawable.ic_menu_share);
+
+// Create a PendingIntent to your BroadCastReceiver implementation
+Intent actionIntent = new Intent(
+       this.getApplicationContext(), ShareBroadcastReceiver.class);
+PendingIntent pendingIntent =
+       PendingIntent.getBroadcast(getApplicationContext(), 0 /* request code */, actionIntent, PendingIntent.FLAG_MUTABLE);           
+
+//Set the pendingIntent as the action to be performed when the button is clicked.
+CustomTabsIntent intentBuilder = new CustomTabsIntent.Builder()
+    …
+    .setActionButton(shareIcon, shareDescription, pendingIntent)
+    .build();
+```
+
+## Add custom menu items
+
+Custom Tabs will have a set of default actions provided by the browser. Those actions can include items like "Forward", "Page Info", "Refresh", "Find in Page" or "Open in Browser".  You can add up to five menu items that will appear in the menu below the icon row and above the browser specific menu items. 
+
+You can access your custom actions via the three dot menu in the top right corner:
+
+{% Img src="image/6hHqS5auVgWhN0cQNQztaJx5w4M2/3WXYfUUMRMqzNwgIT1Ao.png", alt="Custom Tab with 5 custom menu items.", width="320", height="693", class="screenshot screenshot--filled" %}
+
+
+A menu item is added by calling [CustomTabsIntent.Builder#addMenuItem](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder#addMenuItem(java.lang.String,android.app.PendingIntent)) with title and a [PendingIntent](https://developer.android.com/reference/android/app/PendingIntent) that browser will call on your behalf when the user taps the item are passed as parameters.
+
+```java
+CustomTabsIntent intent = new CustomTabsIntent.Builder()
+        ...
+        .addMenuItem("Menu item 1", pendingIntent)
+        .addMenuItem("Menu item 2", pendingIntent)
+        .addMenuItem("Menu item 3", pendingIntent)
+        .addMenuItem("Menu item 4", pendingIntent)
+        .addMenuItem("Menu item 5", pendingIntent)
+        .build();
+```
+
+## Customize the close button
+
+Customizing the close button can be a meaningful way to better fit a Custom Tab into the flow of your app. If you want the user to feel like Custom Tabs is a modal dialog, use the default `“X”` button. If you want the user to feel the Custom Tab is part of the application flow, use the back arrow.
```

Comment:
> Customizing the close button can be a meaningful way to better fit a Custom Tab into the flow of your app

the tone is odd compared to the rest of the doc, maybe 

> Customizing the close button helps {{integrate|blend}} your Custom Tab into the flow of your app

Reply:
rephrased